src/components/molecules/AnnouncementMessage/AnnouncementMessage.tsx

```diff
@@ -14,42 +14,42 @@ import { LinkButton } from "components/atoms/LinkButton";
 import "./AnnouncementMessage.scss";
 
 export interface AnnouncementMessageProps {
-  banner?: Banner;
```

Comment:
why was that changed? looks like we could get banner from everywhere without the need to get it here manually,

Reply:
I suppose it would be better to fetch banner info inside the banner component itself, so that it doesn't rely on the parent component